scripts/slurm_runner.py

```diff
@@ -212,47 +254,302 @@ def launch_docker(infra_dir, docker_home, available_nodes, node=None, dbg_lvl=1)
+
+    # Check each log file for errors
+    for file in log_files:
+        with open(root_directory+file, 'r') as f:
+            contents = f.read()
+            split = contents.split(" ")
+
+            # Cannot get config if file isn't complete
+            if len(split) < 2:
+                continue
+
+            config = split[1] 
+
+            # Check if currently running, skip if so. Running simulations will not contain
+            # the completion message
+            if "stat_collection_job" not in file:
+                # Non-stat jobs will have 4 lines in their log file until they complete.
+                # Fifth line completion message indicates completion
+                if len(contents.split("\n")) < 5:
+                    skipped += 1
+                    running[config] += 1
+                    continue
             else:
-                print(f"\033[31mUNAVAILABLE: {node}\033[0m")
+                # Stat jobs were modified to print DONE as a final message
+                if "DONE" not in contents:
+                    # skipped += 1
+                    stats_generating = True
+                    continue
 
-    print(f"\nChecking what nodes have a running container with name {docker_prefix}_*_{job_name}_*_*_{user}")
-    node_docker_running = check_docker_container_running(available_slurm_nodes, docker_prefix_list, job_name, user, dbg_lvl)
 
-    for node in all_nodes:
-        if node in node_docker_running.keys() and len(node_docker_running.get(node)) > 0:
-            print(f"\033[92mRUNNING:     {node}\033[0m")
-            for docker in node_docker_running[node]:
-                print(f"\033[92m    CONTAINER: {docker}\033[0m")
-        else:
-            print(f"\033[31mNOT RUNNING: {node}\033[0m")
+            # Most scarab runs and all stat runs will have a line with "Error" in them if they fail
+            if 'Error' in contents:
+                error_runs += [root_directory+file]
+                failed[config] += 1
+                continue
+
+            # To be sure, check scarab runs with for final success line
+            if descriptor_data["experiment"] not in contents:
+                error_runs += [root_directory+file]
+                failed[config] += 1
+                continue
 
+            if config != 'stat':
+                completed[config] += 1
+    
+
+    print(f"Currently running {len(running_sims)} simulations (from logs: {skipped})")
+    if stats_generating:
+        print("Stat collector is running")
+
+    # print(f"\033[92mSuccessfully Completed Jobs: {len(all_jobs) - len(not_complete) - len(error_runs)}\033[0m")
+    
+    data = {"Configuration":[],"Completed":[],"Failed":[],"Running":[],"Pending":[],"Total":[]}
+    for conf in confs:
+        data["Configuration"].append(conf)
+        data["Completed"].append(completed[conf])
+        data["Failed"].append(failed[conf])
+        data["Running"].append(running[conf])
+        data["Pending"].append(pending[conf])
```

Comment:
Can you add a column "Nonexisting" for all runs that have never been run (dir does not exist)
Can you separate Failed into "Failed" and "Slurm-error"